backend/slackbot/worker.js

```diff
@@ -117,7 +123,12 @@ async function handleCommand(slackFields){
   // Post messages to slack
   for (let message in slackResponses) {
     slackResponses[message].response_type = 'in_channel';
-    await axios.post(responseURL, slackResponses[message]);
+
+    try {
+      await axios.post(responseURL, slackResponses[message]);
```

Comment:
When something fails on Slack's end (authentication failure, etc.), it should send back a non-200 HTTP response. Those won't trigger the try/catch, so make sure you're checking for those cases separately and handling them.